lib/cli.rb

```diff
@@ -920,6 +920,8 @@ def self.parse_inspect_command_options(host, options)
 
       if options[:public]
         ip = "0.0.0.0"
+
+        Hint.print(:serve_public_option)
```

Comment:
I think this should rather be a warning instead of a hint so that it is shown even if the user disabled the hints.